packages/wallet-adapter-core/src/utils/types.ts

```diff
@@ -25,15 +25,17 @@ export interface AptosStandardSupportedWallet {
 
 // Update this with the name of your wallet when you create a new wallet plugin.
 export type AvailableWallets =
+  | "Continue with Apple"
+  | "Continue with Google"
+  | "Leap Wallet"
```

Comment:
`Leap wallet` is not supported by the adapter, please remove

Reply:
Do you know if Rimosafe is supported?

Reply:
@0xmaayan - I've removed Leap wallet. I've left Rimosafe. Let me know if you want me to remove that too.